homeassistant/components/imeon_inverter/coordinator.py

```diff
@@ -0,0 +1,131 @@
+"""Coordinator for Imeon integration."""
+
+from __future__ import annotations
+
+from asyncio import timeout
+from datetime import timedelta
+import logging
+from typing import Any
+
+from imeon_inverter_api.inverter import Inverter
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import HUBNAME, TIMEOUT
+
+_LOGGER = logging.getLogger(__name__)
+
+type InverterConfigEntry = ConfigEntry[InverterCoordinator]
+
+
+# HUB CREATION #
+class InverterCoordinator(DataUpdateCoordinator[dict[str, Any]]):
```

Comment:
```suggestion
class InverterCoordinator(DataUpdateCoordinator[dict[str, Any]]):
```
can we type that more precisely than `dict[str, Any]`?

Reply:
how ?

Reply:
By adding the type of whatever you put in the dictionary